util/app-config/src/sentry_config.rs

```diff
@@ -31,8 +27,6 @@ impl SentryConfig {
                     scope.set_extra("org_contact", org_contact.clone().into());
                 }
             });
-
-            register_panic_handler();
```

Comment:
How the sentry panic handler is registered in the new version?

Reply:
[`Sentry` will set itself as the panic handler automatically on setup, so no need to do that manually](../../../../getsentry/sentry-rust/issues/235#issuecomment-654832365)

Reply:
OK. I'm verifying it.

Reply:
http://sentry.nervos.org/share/issue/e7395d8f2f964df3b5d3ef2139626680/